src/v2/providers/alerts/appDistribution.ts

```diff
@@ -53,6 +53,8 @@ export interface NewTesterDevicePayload {
 /**
  * The internal payload object for receiving in-app feedback from a tester.
  * Payload is wrapped inside a `FirebaseAlertData` object.
+ *
+ * @internal
```

Comment:
I'd consider using `@alpha`.

`@internal`: typescript tag. Removes the function entirely out of build output.
`@alpha`: api-extractor tag used for reference doc generation. Keeps the function in build output allowing us to test it. No reference doc will be generated

Reply:
Thanks for the tip! Done.